test/unit/mdc-drawer/mdc-drawer.test.js

```diff
@@ -104,35 +76,34 @@ test('#set open false calls foundation.close', () => {
 });
 
 test('keydown event calls foundation.handleKeydown method', () => {
-  const {component} = setupTest();
+  const {component, drawer} = setupTest();
   component.foundation_.handleKeydown = td.func();
-  component.initialSyncWithDOM();
-  domEvents.emit(document, 'keydown');
+  drawer.querySelector('.mdc-list-item').focus();
+  domEvents.emit(drawer, 'keydown');
   td.verify(component.foundation_.handleKeydown(td.matchers.isA(Object)), {times: 1});
 });
 
 test('transitionend event calls foundation.handleTransitionEnd method', () => {
   const {component, drawer} = setupTest();
   component.foundation_.handleTransitionEnd = td.func();
-  component.initialSyncWithDOM();
   domEvents.emit(drawer, 'transitionend');
   td.verify(component.foundation_.handleTransitionEnd(td.matchers.isA(Object)), {times: 1});
 });
 
 test('#destroy removes keydown event listener', () => {
-  const {component} = setupTest();
-  const originalRemove = document.removeEventListener;
-  document.removeEventListener = td.func();
+  const {component, drawer, mockFoundation} = setupTest();
   component.destroy();
-  td.verify(document.removeEventListener('keydown', td.matchers.isA(Function)), {times: 1});
-  document.removeEventListener = originalRemove;
+
+  domEvents.emit(drawer, 'keydown');
```

Comment:
Does this need a focus call first to be consistent with the other keydown test? (i.e. would this not even call handleKeydown even if it weren't destroyed?)

Reply:
Done.